rust_dev_preview/examples/qldb/src/bin/list-ledgers.rs

```diff
@@ -28,7 +27,7 @@ async fn show_ledgers(client: &QLDBClient) -> Result<(), Error> {
 
     while let Some(page) = pages.next().await {
```

Comment:
I think you want `try_next` here—this is unwrapping a potential error from the service and could panic

Reply:
Probably? But I don't know what this code was trying to do in the first place. Because it has an explicit `page_size` I think Doug was meaning to show using the paginator itself? Or it was before .items existed? Opened https://github.com/awsdocs/aws-doc-sdk-examples/issues/5448 to track